influxdb/src/main/scala/akka/stream/alpakka/influxdb/scaladsl/InfluxDbFlow.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.influxdb.{impl, InfluxDbWriteMessage, InfluxDbWriteResult, InfluxDbWriteSettings}
+import akka.stream.scaladsl.Flow
+import org.influxdb.InfluxDB
+import org.influxdb.dto.Point
+
+import scala.collection.immutable
+
+/**
+ * Scala API to create InfluxDB flows.
+ */
+object InfluxDbFlow {
+
+  def create(settings: InfluxDbWriteSettings)(
+      implicit influxDB: InfluxDB
+  ): Flow[InfluxDbWriteMessage[Point, NotUsed], InfluxDbWriteResult[Point, NotUsed], NotUsed] =
+    Flow[InfluxDbWriteMessage[Point, NotUsed]]
+      .batch(settings.batchSize, immutable.Seq(_))(_ :+ _)
+      .via(new impl.InfluxDbFlowStage[NotUsed](influxDB))
+      .mapConcat(identity)
+
+  def typed[T](clazz: Class[T], settings: InfluxDbWriteSettings)(
+      implicit influxDB: InfluxDB
+  ): Flow[InfluxDbWriteMessage[T, NotUsed], InfluxDbWriteResult[T, NotUsed], NotUsed] =
+    Flow[InfluxDbWriteMessage[T, NotUsed]]
+      .batch(settings.batchSize, immutable.Seq(_))(_ :+ _)
+      .via(new impl.InfluxDbMapperFlowStage[T, NotUsed](clazz, influxDB))
+      .mapConcat(identity)
+
+  def createWithPassThrough[C](settings: InfluxDbWriteSettings)(
```

Comment:
Alternatively, just offer the pass-through versions and leave it to the user to set the `NotUsed` type. It shows anyway in the elements.